virttest/qemu_devices/qcontainer.py

```diff
@@ -3105,16 +3105,24 @@ def _emulator_setup(binary, extra_options=None):
             if tpm_version in ('2.0',):
                 setup_cmd += ' --tpm2'
 
-            tpm_path = os.path.join(tmp_dir, '%s_%s_tpm_state_%s' %
-                                    (self.vmname, name, random_str))
-            if os.path.exists(tpm_path):
-                shutil.rmtree(tpm_path, ignore_errors=True)
-            os.makedirs(tpm_path)
-            setup_cmd += " --tpm-state %s" % tpm_path
+            tpm_state_path = params.get(
+                "tpm_state_path", os.path.join(
+                    tmp_dir, '%s_%s_tpm_state_%s' % (
+                        self.vmname, name, random_str)))
+            if not os.path.exists(tpm_state_path):
+                os.makedirs(tpm_state_path)
+            setup_cmd += " --tpm-state %s" % tpm_state_path
 
             setup_cmd += (" --createek --create-ek-cert"
                           " --create-platform-cert"
-                          " --lock-nvram --not-overwrite")
+                          " --lock-nvram")
+            tpm_overwrite = params.get("tpm_overwrite")
```

Comment:
tpm_overwrite is a required parameter for tpm tests? If it is not set, the overwrite_option will not be added ?

Reply:
Please see the following code, default is `None` and will not be added.

Reply:
I mean if we want to add this option in the command, we need configure it in test case or command line. Previously the default value is not None.

Reply:
`--overwrite` and `--not-overwrite` are only useful when the TPM device exists, I don't think it's a required option, but if you want to set a default value to avoid some issues, it's also okay for me.